src/app/components/app-layout/app-layout.component.html

```diff
@@ -28,10 +28,6 @@
       <a routerLink="project/overview" routerLinkActive="active" class="nav-link nav-text">Projects</a>
       <a routerLink="project/add/source" routerLinkActive="active" class="nav-link nav-text">Add project</a>
     </div>
-    <div class="search-bar">
```

Comment:
Maybe we should hide these elements instead of removing them from the code

Reply:
That's what I was thinking as well, I had a discussion with the frontend team though and according to user tests, this should be removed for now.

Reply:
Based on what? Changing a CSS property to display the element is easier than going back to this PR in the future and adding the elements back to the codebase.

Reply:
They've performed user tests, users did not see added value in this being there. This is why it will be removed from the design completely and won't be added back for now. I agree that for things that have not been implemented yet but are in the design, we can hide them. However, if it's no longer in the design, we should not hide it.